sdk/storage/Azure.Storage.Files.Shares/src/ShareFileClient.cs

```diff
@@ -714,6 +714,7 @@ private void SetNameFieldsIfNull()
                             fileAttributes: smbProps.FileAttributes?.ToAttributesString() ?? Constants.File.FileAttributesNone,
                             fileCreationTime: smbProps.FileCreatedOn.ToFileDateTimeString() ?? Constants.File.FileTimeNow,
                             fileLastWriteTime: smbProps.FileLastWrittenOn.ToFileDateTimeString() ?? Constants.File.FileTimeNow,
+                            fileChangeTime: smbProps.FileChangedOn.ToFileDateTimeString(),
```

Comment:
Would this get set to `default` if there's no value? Just making sure it doesn't get set to something weird like `DateTimeOffset.MinValue` or something.

Reply:
This parameter is optional, with a default value set to the time of the request time.  [Spec](https://microsoft.sharepoint.com/:w:/t/AzureStorage/EWulSSFYq0hEuvhA2RMI0IgBdYZiKrpALCxYXZJ5P7nXzQ) for more context.